politeiawww/comments.go

```diff
@@ -131,29 +146,50 @@ func (p *politeiawww) updateCommentScore(token, commentID string) (int64, error)
 		case prevAction == 0:
 			// No previous action so we add the new action to the
 			// vote score
-			score += action
+			negativeVote, avv := absVoteValue(action)
```

Comment:
This is still unnecessarily complex.  A vote can only be either 1 or -1.  You don't need a function for this.  Just check it directly and take the appropriate action.  Its much easier to read this way.  Ex.

```
			// No previous action so we add the new action to the
			// vote score
			switch action {
			case -1:
				votes.down += 1
			case 1:
				votes.up += 1
			}
```

Reply:
Done. Sorry it took so many iterations to get to this point -- still figuring out how you folks operate :-P

Reply:
Much better!  Last nit, return counters as a pointer from updateCommentVotes so that you can return nil on errors instead of counters{}.

Reply:
Done.